weather/www/js/cityController.js

```diff
@@ -0,0 +1,107 @@
+// Ionic Weather App - City Controller
+
+'use strict';
+
+var weatherApp = angular.module('weather')
+
+
+
+.run(function($ionicPlatform) {
+  $ionicPlatform.ready(function() {
+    if(window.cordova && window.cordova.plugins.Keyboard) {
+      // Hide the accessory bar by default (remove this to show the accessory bar above the keyboard
+      // for form inputs)
+      cordova.plugins.Keyboard.hideKeyboardAccessoryBar(true);
+
+      // Don't remove this line unless you know what you are doing. It stops the viewport
+      // from snapping when text inputs are focused. Ionic handles this internally for
+      // a much nicer keyboard experience.
+      cordova.plugins.Keyboard.disableScroll(true);
+    }
+    if(window.StatusBar) {
+      StatusBar.styleDefault();
+    }
+  });
+})
+
+.factory('Cities', function() {
+  return {
```

Comment:
I'm guessing these are for some functionality that is currently unfinished? All good but could be worth mentioning